src/components/tags/styled-td.tsx

```diff
@@ -1,7 +1,5 @@
-import styled from 'styled-components'
+import { PropsWithChildren, FC } from 'react'
 
-export const StyledTd = styled.td`
-  padding: 10px;
-  vertical-align: top;
-  border: 3px solid ${(props) => props.theme.colors.lightBlueBackground};
-`
+export const StyledTd: FC<PropsWithChildren<{ className?: string }>> = (
```

Comment:
@Entkenntnis Experiment. Basically just shortens `<td className="serlo-td">` to `<StyledTd>`

Reply:
Styled Components is doing a lot of hidden magic to make this approach work and make the component behave like a regular dom element, e.g. ref forwarding, display name, props forwarding, ...

While this is doable, I do not yet know if this is worth the effort of building such a wrapper.

Reply:
Okay, removed for now.